proposal.Rmd

```diff
@@ -0,0 +1,22 @@
+---
+title: "Proposal: shades of foundation offered by a variety of beauty brands around the world"
+output: github_document
+---
+
+# Section 1: Motivation and Purpose
+
+**Our role**: data scientist consultancy firm
```

Comment:
minor wording change: data science consultancy firm

Reply:
Thank you for the suggestion Ela! I used this cus this is how was written in milestone 1 instruction - will double check with Joel if this is a typo!